### PR TITLE
 Finalize Kurdish translation

### PR DESCRIPTION
**Pull request by CYAXXX**

### CODE REVIEW

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -832,11 +840,6 @@ msgctxt "precise_user_location"
 msgid "Cancel"
 msgstr "Têk bibe"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:842]:
```suggestion

#. Indicator that a game has been cancelled
msgctxt "Sports module"
msgid "Cancelled"
msgstr ""
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -806,7 +809,12 @@ msgid ""
 "permanent way (on a remote server in the cloud). No personally identifiable "
 "information will be stored in the cloud, and your passphrase will never "
 "leave your browser."
-msgstr ""
+msgstr "Bi berdestî, sazkarî têne tomarkirin di kilorên gerokê yên ne taybet (di "
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:812]:
```suggestion
msgstr ""
"Bi berdestî, sazkarî têne tomarkirin di kilorên gerokê yên ne taybet (di "
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -1499,18 +1503,13 @@ msgstr "Berdest"
 msgid "Definition"
 msgstr "Pênase"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1505]:
```suggestion

#. Indicator that a game has been delayed
msgctxt "Sports module"
msgid "Delayed"
msgstr ""
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -1912,12 +1909,7 @@ msgstr "Pêvek & Bêtir"
 
 msgctxt "size"
 msgid "Extra Large"
-msgstr "Pir mezin"
-
-#. Title for the FIFA World Cup. Please keep it short and generic (without year nor location), and keep FIFA in it.
-msgctxt "Sports module"
-msgid "FIFA World Cup"
-msgstr ""
+msgstr "Pir Mezin"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1912]:
```suggestion
msgstr "Pir Mezin"

#. Title for the FIFA World Cup. Please keep it short and generic (without year nor location), and keep FIFA in it.
msgctxt "Sports module"
msgid "FIFA World Cup"
msgstr ""
```

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 2 above)
Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po:812]: ya dizanî? li update ya dawî binêre berî çend hefteyên berê di wir de pirsgirêk tune bû bo ev hevoka dirêj, çavekê li wir bixe baştir dibe

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 5]
```diff
@@ -2612,121 +2620,118 @@ msgid "Indonesia"
...
 
 #. Title for one of the <a href="https://duckduckgo.com/params.html">URL params page section</a>.
 msgid "Interface Settings"
-msgstr ""
-
-#. Indicator that a game has been interrupted
-msgctxt "Sports module"
-msgid "Interrupted"
-msgstr ""
+msgstr "Sazkariyên Navrûyê"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:2685]:
```suggestion
msgstr "Sazkariyên Navrûyê"

#. Indicator that a game has been interrupted
msgctxt "Sports module"
msgid "Interrupted"
msgstr ""
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 6]
```diff
@@ -2846,12 +2857,12 @@ msgstr "Ziman"
 #. Text of a tooltip displayed when the user has swapped origin and translated languages
 msgctxt "translations_module"
 msgid "Languages swapped"
-msgstr ""
+msgstr "Ziman hatin guhertin"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:2860]:
```suggestion
msgstr "Ziman hate guhertin"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 7]
```diff
@@ -5222,11 +5265,6 @@ msgstr "Pir fereh"
 msgid "Support"
 msgstr "Alîkarî"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:5267]:
```suggestion

#. Indicator that a game has been suspended
msgctxt "Sports module"
msgid "Suspended"
msgstr ""
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 8]
```diff
@@ -4071,54 +4118,49 @@ msgctxt "language_name"
 msgid "Portuguese (Portugal)"
 msgstr "Portûgalî (Portûgal)"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:4120]:
```suggestion

#. Indicator that a game has been postponed
msgctxt "Sports module"
msgid "Postponed"
msgstr ""
```

**Review by roj1512 — APPROVED** [review]
Thank you so much for your countless efforts, @CYAXXX!

**Comment by roj1512** [comment]
@GioSensation Are we ready to ship this? :)

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 9]
```diff
@@ -5858,33 +5935,38 @@ msgstr "Viyetnamî"
 msgid "View File"
 msgstr "Pelê Nîşan bide"
 
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:5938]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 10]
```diff
@@ -5858,33 +5935,38 @@ msgstr "Viyetnamî"
...
-msgstr "Li ser Bexşeyê nîşan bide"
+msgstr "Li ser Nexşeyê nîşan bide"
 
 msgctxt "Ads GDPR, internal use only. Do not change"
 msgid ""
 "Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by "
 "Microsoft's ad network"
 msgstr ""
 
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:5952]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 11]
```diff
@@ -6056,14 +6138,19 @@ msgid ""
 "incorporated at the discretion of %s. Corrections may not show up in results "
 "right away."
 msgstr ""
+"Em paşragihandinan wekî vê bi kar tînin bo pêşxistina DuckDuckGo. "
+"Pêşniyar wê di encama serrastkirinê %s de cih bigire. Dibe ku di cih de "
+"encamên wê dernekevin holê."
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:6144]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 12]
```diff
@@ -6146,21 +6233,24 @@ msgid ""
 "What is the Cloud Save bookmarklet and how does it differ from the URL "
 "parameter bookmarklet?"
 msgstr ""
+"Şûnpela Tomarkirina Ewrî çi ye û çawa ji şûnpela parametreyên Girêdanê "
+"cuda dibe?"
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:6238]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 13]
```diff
@@ -6272,73 +6363,82 @@ msgid ""
...
+"Tu niha bi taybetî digerî. %sŞîretên me bi dest bixe bo şopandina hîn kêm bike."
 
 msgid ""
 "YouTube (owned by Google) does not let you watch videos anonymously. As "
 "such, watching YouTube videos here will be tracked by YouTube/Google."
 msgstr ""
+"YouTube (Xwediyê wê Google e) nahêle ku tu bi awayekî nenas vîdyoyan "
+"temaşe bikî. Wekî din temaşekirina vîdyoyên YouTube li vir wê werin "
+"şopandin ji aliyê YouTube/Google."
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:6430]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 14]
```diff
@@ -6462,43 +6570,45 @@ msgstr "mh"
...
-msgstr ""
+msgstr "%s ne kodkirin e"
 
 #. Used for the <em>color settings</em> into the <a href="https://duckduckgo.com/params.html">URL params page</a> as the last available value for the color parameter.
 #. <em>%s</em> is for a hexadecimal color code as <em>#395323</em>, but it has to be safe encoded, and as <em>#</em> seems not safe, <em>%23</em> must be used in place of the sharp symbol, but they are the same for your browser.
 msgid ""
 "or write out the color code you want, e.g. %s (%s is an encoded %s char)."
 msgstr ""
+"an jî koda rengê ku tu dixwazî binivîse, mînak. %s (%s kodek e %s tîp e)."
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:6584]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 15]
```diff
@@ -5941,45 +6023,45 @@ msgstr "Li ser YouTube temaşe bike"
...
 
 msgid ""
 "We don't store your personal info. We don't follow you around with ads. We "
 "don't track you. Ever."
 msgstr ""
-"Em zanyariyên te yên kesane tomar nakin. Em te naşopînin bi riya reklaman. "
-"Em te naşopînin. Ti carî."
+"Em zanyariyên te yên taybet tomar nakin. Em te naşopînin bi riya reklaman. Em "
+"te naşopînin. Ti carî."
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:6060]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 16]
```diff
@@ -5858,33 +5935,38 @@ msgstr "Viyetnamî"
...
 
+
 msgctxt "mobile promotion on desktop"
 msgid ""
 "Visit %sDuckDuckGo.com%s on your tablet or phone and follow the provided "
 "instructions."
 msgstr ""
+"Serdana %sDuckDuckGo.com%s bike li ser tablet an jî mobayîla xwe û Rênasînên "
+"dayî bişopîne."
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:5960]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 17]
```diff
@@ -4671,146 +4730,151 @@ msgstr "Wêneyan Bibîne"
...
 
 msgid "Select %sManage add-ons%s from the dropdown menu"
-msgstr ""
+msgstr "%sPêvekan bi rê ve bibe%s hilbijêre ji menuya daketî"
 
 msgid ""
 "Select %sOpera > Preferences%s (on Mac) or %sMenu > Settings%s (on Windows)"
 msgstr ""
+"%sOpera > Hilbijartin%s (li ser Mac) an jî %sMenu > Sazkarî%s (li ser Windows) hilbijêre"
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:4783]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 18]
```diff
@@ -4671,146 +4730,151 @@ msgstr "Wêneyan Bibîne"
...
-msgstr ""
+msgstr "%sDuckDuckGo%s hilbijêre di menuya daketî ya motora lêgerînê ya Berdest de" 
 
 msgid "Select %sDuckDuckGo%s in the list of search providers"
-msgstr ""
+msgstr "%sDuckDuckGo%s hilbijêre di rêzoka peydekarên lêgerînê de"
 
 msgid "Select %sDuckDuckGo%s under the%sDefault Search Engine%s section"
-msgstr ""
+msgstr "%sDuckDuckGo%s hilbijêre di bin beşa %sMotorên Lêgerînê yên Berdest%s de"
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:4768]:
```suggestion
msgstr "%sDuckDuckGo%s hilbijêre di bin beşa %sMotorên Lêgerînê yên Berdest%s de"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 19]
```diff
@@ -4671,146 +4730,151 @@ msgstr "Wêneyan Bibîne"
...
-msgstr ""
+msgstr "%sDuckDuckGo%s hilbijêre û %sWekî berdest saz bike%s bitikîne"
 
 msgid "Select %sDuckDuckGo%s and click %sSet as default%s."
-msgstr ""
+msgstr "%sDuckDuckGo%s hilbijêre û %sWekî berdest saz bike%s bitikîne."
 
 msgid "Select %sDuckDuckGo%s in the Default Search Engine drop down"
-msgstr ""
+msgstr "%sDuckDuckGo%s hilbijêre di menuya daketî ya motora lêgerînê ya Berdest de"
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:4762]:
```suggestion
msgstr "%sDuckDuckGo%s hilbijêre di menuya daketî ya motora lêgerînê ya Berdest de"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 20]
```diff
@@ -2737,13 +2759,19 @@ msgid ""
 "through Microsoft's Ad Network. Clicks lead directly to merchant landing "
 "pages and unlike ads, DuckDuckGo is not compensated for these results."
 msgstr ""
+"Hêman li gorî lêgerînên te têne rêzkirin û radestkirin bi riya Tora Reklamê "
+"ya Microsoft. Tikandin rasterast dişîne rûpel û reklamê bazirganan, "
+"DuckDuckGo şûndana van ziyanan nake."
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:2765]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 21]
```diff
@@ -2612,121 +2632,123 @@ msgid "Indonesia"
...
 msgid "Israel"
-msgstr "Îsraêl"
+msgstr "Îsraîl"
 
 msgctxt "settings"
 msgid ""
 "It's okay to (very infrequently) ask me about my experience using DuckDuckGo"
 msgstr ""
+"Pirsgirek tune ku (car caran) were pirskirin derbarê tecrubeya min bi DuckDuckGo re"
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:2743]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 22]
```diff
@@ -2523,39 +2527,50 @@ msgid ""
...
 "%sOther Search Engines%s"
 msgstr ""
+"Ku tu %sDuckDuckGo%s nabînî, wê pêdiviya te bi tevlîkirinê wê hebe bo rêzoka "
+"%sMotorên Lêgerînê Yên Din%s"
 
 #. http://i.imgur.com/ReKkJtU.png
 msgctxt "settings"
 msgid "If you still want to support us, %shelp spread DuckDuckGo%s"
-msgstr ""
+msgstr "Ku tu hîn dixwazî piştgiriya me bikî, %salîkariya belavkirina DuckDuckGo bike%s"
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:2554]:
```suggestion
msgstr "Ku tu hîn dixwazî piştgiriya me bikî, %salîkariya belavkirina DuckDuckGo bike%s"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 23]
```diff
@@ -2523,39 +2527,50 @@ msgid ""
...
 msgctxt "precise_user_location"
 msgid ""
 "If the browser location remains unavailable, then in your browser go to %s⋮ "
 "> Menu > Settings > Site settings > Location%s, and ensure DuckDuckGo is "
 "allowed location access."
 msgstr ""
+"Ku cihê gerokê neçalak dimîne, wê demê biçe gerokê xwe %s⋮ "
+"> Menu > Sazkarî > Sazkariyên malperê > Cih%s, û piştrast "
+"bike ku mafê gihiştîna cihê hatiye dayîn bo DuckDuckGo."
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:2542]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 24]
```diff
@@ -1176,8 +1182,9 @@ msgid ""
 "Click the dropdown menu beside %sSearch engine used in the address bar%s and "
 "select %sDuckDuckGo%s."
 msgstr ""
-"Li kêleka %sMotora lêgerînê ya ku di darikê navnîşanê%s de tê bikaranîn "
-"bitikîne û%sDuckDuckGo%s hilbijêre."
+"Li kêleka %sMotora lêgerînê ya ku di darikê navnîşanê de tê bikaranîn%s bitikîne û "
+"%sDuckDuckGo%s hilbijêre."
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1187]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 25]
```diff
@@ -807,6 +810,12 @@ msgid ""
 "information will be stored in the cloud, and your passphrase will never "
 "leave your browser."
 msgstr ""
+"Bi berdestî, sazkarî têne tomarkirin di kilorên gerokê yên ne taybet (di "
+"gerokê te de). Tu dikarî Tomarkirina Ewrî ya nenas bi kar bînî bo ku sazkariyên "
+"xwe tomar bikî bi awayeke mayînde (li ser rajekarekî ji dû re ve di ewrê de). "
+"Ti zanyariyên taybet ku werin naskirin nayên tomarkirin di ewrê de, û "
+"borînpeyva te wê ti carî ji dervyî geroka te dernekevê."
+
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:818]:
```suggestion
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 26]
```diff
@@ -24,12 +24,12 @@ msgstr "!Bang kurteyên lêgerînê"
 #. Heading for the table column indicating what percent of the population is fully vaccinated
 msgctxt "Covid 19 module"
 msgid "% Fully Vaccinated"
-msgstr "% 100% Derzî kirî"
+msgstr "% 100 Deqandî"
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:27]: Can you guys please double check these two strings? They don't seem to match the expectations when using a machine translator.

Comment (roj1512) [reply locales/ku/LC_MESSAGES/duckduckgo.po]: Fixed :)

Comment (GioSensation) [reply locales/ku/LC_MESSAGES/duckduckgo.po]: The machine translation still returns weird translations like "detached" or "decayed", but if you guys are sure the translation is correct I'm happy to proceed.

**Comment by GioSensation** [comment]
Thanks! Merged 🎉.

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 26 above)
Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po]: > The machine translation still returns weird translations like "detached" or "decayed", but if you guys are sure the translation is correct I'm happy to proceed.

@GioSensation Hi, I can see the weird results from different online translation services like  
Google/Bing/Lingvanex. I'll report it btw for a better experience.
Whatever you can check **Deqandî** meanings in[Wikiferheng (**Kurdish version of** **Wiki Dictionary**)](https://ku.m.wiktionary.org/wiki/deqand%C3%AE).
Thank you for mentioning that.

**Pull request merged**